FireRender.Maya.Src/MayaStandardNodesSupport/NodeProcessingUtils.cpp

```diff
@@ -0,0 +1,65 @@
+#include "FireMaya.h"
+#include "BaseConverter.h"
+
+#include "NodeProcessingUtils.h" 
+#include <maya/MRampAttribute.h>
+
+namespace MayaStandardNodeConverters
+{ 
+
+frw::Value GetSamplerNodeForValue(
+	const ConverterParams& params,
+	const MString& plugName,
+	frw::Value valueInput,
+	frw::Value inputMin,
+	frw::Value inputMax,
+	frw::Value outputMin,
+	frw::Value outputMax)
+{
+	MPlug plug = params.shaderNode.findPlug(plugName, false);
+
+	if (plug.isNull())
+	{
+		assert(!"RemapHSV parse. Error: could not get the plug");
+		return frw::Value();
+	}
+
+	const unsigned int bufferSize = 256; // same as in Blender
+
+	// create buffer desc
```

Comment:
Consider moving this into a function

Reply:
Discussed. Will be refactored later if its really becomes needed